packages/asc-ui/src/components/Button/ButtonStyle.ts

```diff
@@ -232,7 +232,8 @@ const ButtonStyle = styled.button<Props>`
   font-size: 16px;
   font-weight: 700;
   line-height: 20px;
-  padding: ${({ size: sizeProp }) => (sizeProp ? '0' : '12px 15px')};
+  padding: ${({ size: sizeProp, theme }) =>
+    sizeProp ? '0' : themeSpacing(3, 4)({ theme })};
```

Comment:
Do we need to pass the theme here? It should be implicit or perhaps:

```suggestion
  padding: ${({ size }) => size ? '0' : css`${themeSpacing(3, 4)`};
```